css/styles.css

```diff
@@ -231,3 +237,29 @@ button {
   bottom: 0;
   z-index: -1;
 }
+
+.back-buttons {
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.back, .back2 {
+  border-radius: 15px;
+  color: #3D3D3D;
+  background-color: #da5552;
+  padding: 5px 5px;
+}
```

Comment:
In general, we don't put numbers on classes like this. What is the difference between `.back` and `.back2`? Make sure it's easy to tell by reading your code.